boschshcpy/services_impl.py

```diff
@@ -622,6 +622,20 @@ def summary(self):
         print(f"    acousticSignalState      : {self.acousticSignalState}")
 
 
+class WaterLeakageSensorCheckService(SHCDeviceService):
+    class State(Enum):
+        NONE = "NONE"
+
+    @property
+    def value(self) -> State:
+        return self.State(self.state["result"])
```

Comment:
Parsing the state of the sensor will result in an API exception when the self.state["result"] is not "NONE". If only the state string is used, error handling can be done when evaluating the actual state.

Reply:
Could not find any description for `/devices/<device_id>/services/WaterLeakageSensorCheck/state`. I changed the type to string until I find a way to fetch all available enum values.

Reply:
You can either file an issue in the official Bosch API for updating the description, or just watch on state changes when putting the sensor into different modes (e.g. put water on the contacts and just watch how the device behaves in postman).